docs/ibc/proto-docs.md

```diff
@@ -311,11 +311,12 @@ An InterchainAccount is defined as a BaseAccount & the address of the account ow
 <a name="ibc.applications.interchain_accounts.v1.ActiveChannel"></a>
 
 ### ActiveChannel
-ActiveChannel contains a pairing of port ID and channel ID for an active interchain accounts channel
+ActiveChannel contains a connection ID, port ID and associated active channel ID
```

Comment:
In some places we use the terminology `connectionID` & others `connection sequence`. When should we use one or the other? cc @AdityaSripal @colin-axner

Reply:
Anywhere the connection ID is used in the string format `"connection-{sequence}"` eg. `"connection-0"`, then `connectionID` is used, otherwise `connectionSeq` when it's just the parsed uint64 sequence value associated with the connection ID.